src/OpenTelemetry.Api/Trace/TelemetrySpan.cs

```diff
@@ -303,6 +304,64 @@ public TelemetrySpan AddBaggage(string key, string value)
             return this;
         }
 
+        /// <summary>
+        /// Record Exception.
+        /// </summary>
+        /// <param name="ex">Exception to be recorded.</param>
+        /// <returns>The <see cref="TelemetrySpan"/> instance for chaining.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public TelemetrySpan RecordException(Exception ex)
+        {
+            Dictionary<string, object> attributes = new Dictionary<string, object>
```

Comment:
It should bail if `ex` is `null` per symmetry with the other signature.

Reply:
i was thinking if i would add a null check. so thats a yes.